### ISSUE TITLE
Where can I find the timer feature?

### ISSUE THREAD

**Issue by targetnull** [hybridNeo/secure_quiz#1]
Hi,
I see you write in the description that you have added timer to your test, but I couldn't find relevant code in your code. Could you please show me where I can find them to let me know what I have missed?

Thanks.

**Comment by hybridNeo** [comment]
Look at line 185 of
https://github.com/hybridNeo/secure_quiz/blob/master/ta/ta.c

**Comment by targetnull** [comment]
Thanks, but I had thought you added a counter, maybe I misunderstood the word 'timer'.
Do you have any idea how to add a counter to do periodic work?

**Comment by hybridNeo** [comment]
What do you mean by counter , something like SIGALRM ?

**Comment by targetnull** [comment]
Yes, such as displaying a string every 5 minutes.

**Comment by hybridNeo** [comment]
Oh that is threading, as far as I know that has not been implemented.

**Comment by targetnull** [comment]
I see, can I directly intercept the interrupt to achieve it?

**Comment by hybridNeo** [comment]
Yes that seems interesting, let me know if you figure something out, Even I
wanted a feature like that.

**Comment by targetnull** [comment]
Thanks,  I will let you know ASAP if I find something useful.
